streaming/src/test/scala/org/apache/spark/streaming/scheduler/ExecutorAllocationManagerSuite.scala

```diff
@@ -83,12 +96,26 @@ class ExecutorAllocationManagerSuite extends TestSuiteBase
             Map.empty)}
       }
 
-      /** Verify that a particular executor was killed */
+      /** Verify that a particular executor was scaled down. */
       def verifyKilledExec(expectedKilledExec: Option[String]): Unit = {
```

Comment:
Do you want to rename the method to verifyScaledDownExec ? to make it match the comment change ?

Reply:
Sure, I can do that.